.github/workflows/swap.yml

```
@@ -1,18 +1,17 @@
-name: Swap feature tests
+name: Swap functional tests
 
 on:
   workflow_dispatch:
   push:
     branches:
     - main
+    - develop
   pull_request:
 
 jobs:
   job_functional_tests:
-    uses: functori/app-exchange/.github/workflows/reusable_swap_functional_tests.yml@ebc5932d939e2bad5eaf3d5e316626169da6ff23
+    uses: LedgerHQ/app-exchange/.github/workflows/reusable_swap_functional_tests.yml@develop
     with:
-      repo_for_exchange: 'functori/app-exchange'
-      branch_for_exchange: 'develop'
       branch_for_tezos: ${{ github.ref }}
-      run_for_devices: '["nanos", "nanosp", "stax"]'
-      test_filter: '"tezos_new"'
+      repo_for_tezos: ${{ github.repository }}
+      test_filter: '"XLM or xlm or Tezos or tezos"'
```

```suggestion
      test_filter: '"XTZ or xtz or Tezos or tezos"'
```
XLM was probably a bad copy paste from Stellar.
Even though I don't think this enable more test...